rust/opendp_tooling/src/proven/filesystem.rs

```diff
@@ -85,20 +85,28 @@ pub fn make_proof_link(relative_path: &str) -> Result<String> {
     if !absolute_path.exists() {
         return Err(Error::custom(format!("{absolute_path:?} does not exist!")));
     }
-
-    let version = env!("CARGO_PKG_VERSION");
-    let target = if version == "0.0.0+development" {
-        (absolute_path.to_str())
-            .ok_or_else(|| Error::custom("absolute path is empty"))?
-            .to_string()
+    
+    // link from sphinx and rustdoc to latex
+    let proof_uri = if let Ok(local_uri) = env::var("OPENDP_LOCAL_DOCS_URI") {
+        format!("{local_uri}/rust/src")
```

Comment:
I wanted there to be a universal way to enable local docs across latex, sphinx, rustdocs... This couldn't be a feature, because latex can't read rust features. Latex also needs more information. So OPENDP_LOCAL_DOCS_URI is supposed to be a global option. But I couldn't get absolute paths working in rendered pdfs, so the local docs aren't... universally local. It might end up being easier to set up a local docs server for the latex links.